libs/database/src/lib/entities/delayed-job.entity.ts

```diff
@@ -44,5 +62,56 @@ export class DelayedJob extends Model<DelayedJob> {
 
   @Column(BOOLEAN)
   isAcknowledged: boolean;
-  
+
+  @AllowNull
+  @Column(STRING)
+  name: string | null;
+
+  @AllowNull
+  @Column(STRING)
+  entityType: string | null;
+
+  @ForeignKey(() => Project)
+  @ForeignKey(() => Site)
+  @Column
+  entityId: number;
+
+  @BelongsTo(() => Project, {
+    foreignKey: "entityId",
+    constraints: false,
+    scope: {
+      entityType: "App\\Models\\V2\\Projects\\Project"
+    },
+    as: "entityProject"
+  })
+  entityProject?: Project;
+
+  @BelongsTo(() => Site, {
+    foreignKey: "entityId",
+    constraints: false,
+    scope: {
+      entityType: "App\\Models\\V2\\Sites\\Site"
+    },
+    as: "entitySite"
+  })
+  entitySite?: Site;
+
+  declare getEntityProject: () => Promise<Project | null>;
+  declare getEntitySite: () => Promise<Site | null>;
```

Comment:
I don't think these are super useful. With this TS extension, `this.$get('entityProject')` comes out typed correctly.

Reply:
removed